### ISSUE TITLE
disable option for registering multiple times for the same events with one e-mail

### ISSUE THREAD

**Issue by minkah1** [eventespresso/event-espresso-core#4172]
At Arnes we organize a free, 3-day conference with many lectures and workshops.

This conference is organized for teachers who receive a certificate of attendance at the end of the conference. To obtain this certificate, they must register through the single sign of one. At the same time, this conference is also organized for researchers who do not have the option of single sign one, so they are not logged into Wordpress when registering and do not need this type of certificate.

To avoid confusion regarding duplicate applications, we would like to disable option for our users from registering multiple times for the same events with one e-mail. So that only one registration with one email is possible.

**Comment by garthkoyle** [comment]
> To avoid confusion regarding duplicate applications, we would like to disable option for our users from registering multiple times for the same events with one e-mail. So that only one registration with one email is possible.

So, in this case you would be using the WordPress User Integration, with the setting to require users to be logged-in before being allowed to register. 

Additionally, there could be an additional setting to only allow one registration per WordPress user for an event?

**Comment by garthkoyle** [comment]
So, in this case, login cannot be required, but the same email address cannot be used more than once?

**Comment by garthkoyle** [comment]
Also, would someone be allowed to purchase multiple tickets at the same time (say 10 different tickets or time slots) or would they only be allowed to ever get one ticket from that event ever?

**Comment by minkah1** [comment]
That's right, no login is required and the same email address cannot be used more than once. No one should be able to buy multiple tickets for the same time slot, but should be able to buy one ticket for each time slot.
